src/middleware/router.coffee

```diff
@@ -166,6 +192,7 @@ buildNonPrimarySendRequestPromise = (ctx, route, options, path) ->
 
     ctx.routes = [] if not ctx.routes
```

Comment:
there are some router tests that rely on this, there is no easy way to check if the router can route to multiple routes without checking the database, and doing that would replicate an end to end test